docs/Developers/Git_Basics.rst

```diff
@@ -163,6 +163,19 @@ Now that your message is written, you will have to save the file. Press escape
 to exit insert mode, and save the file (in Vim that is being done by pressing
 shift + Z twice).
 
+Run Coala
+------------------
+
+Now you can check if your commit messages and the code formattings
+conform with community guidelines.
+If something goes wrong, coala will let you know. If this fails,
+continuous integration (CI) also fails. It means that you cannot proceed
```

Comment:
Can we write `The continuous integration (CI) will fail if coala reports errors.` instead of `If this fails, continuous integration (CI) also fails`